src/dbnode/environment/config_test.go

```diff
@@ -24,6 +24,8 @@ import (
 	"testing"
 	"time"
 
+	"gopkg.in/yaml.v2"
+
```

Comment:
This should be grouped with the imports at the bottom (for external imports).  Again wish linter was working correctly.